src/Developer/Encryption/FieldLevelEncryption.php

```diff
@@ -324,7 +324,7 @@ private static function toJsonString($object) {
         if (is_null($object)) {
             throw new \InvalidArgumentException('Can\'t get a JSON string from a null object!');
         }
-        if (!is_object($object) && !is_array($object)) {
+        if (is_string($object)) {
             return $object;
```

Comment:
Booleans would've gone through here and be converted to an empty string in `sanitizeJson`